src/systems/kinetic_energy_monitor/KineticEnergyMonitor.hh

```diff
@@ -97,9 +97,10 @@ namespace systems
       </plugin>
     </model>
   \endverbatim */
-  class IGNITION_GAZEBO_VISIBLE KineticEnergyMonitor:
-    public System,
+  class IGNITION_GAZEBO_VISIBLE KineticEnergyMonitor
+  : public System,
```

Comment:
Unrelated change

Reply:
I made some format changes, following the rest of the IGN plugins. Should that be another PR?

Reply:
I'm ok with it, if it was intended.